app/assets/javascripts/map/helpers/guides.js

```diff
@@ -159,7 +146,7 @@ define([
 
       // Layers module
       {
-        selector: "#layersnav-forest-change", 
```

Comment:
I agree with this, I'm also not a fan of trailing white space. Maybe we should try something like http://editorconfig.org ?